app/views/themes/institutional_repository/hyrax/homepage/_recent_document.html.erb

```diff
@@ -5,4 +5,4 @@
   <div class="recent-doc-title">
     <%= link_to recent_document.title_or_label, generate_work_url(recent_document, request) %>
   </div>
-</div>
+<% end %>
```

Comment:
this partial opens 2 dives but only closes one. also, I dont see what is "end"ing here